linux_os/guide/system/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillog/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: sle12,ubuntu2004,ubuntu2204
+prodtype: sle12,sle15,ubuntu2004,ubuntu2204
```

Comment:
Hello Jan,

as mentioned in the past, there is no CCE Number yet for this configuration. CIS requires it for years now, but it seems that it was overlooked during CCE creation. Or at least it is not part of the publicly available Excel sheet provided on ncp.nist.gov/cce.

I saw, that SUSE Maintenaners were requested to review, but I did not see any comments yet.

Since I don't know who the members of this group are I'd ask you to get in touch with them.

Kind regards
Nico

Reply:
Hi @truzzon for SLE15 you can use   CCE-92576-8  , just make sure that in the context of the same PR you remove those ID from file shared/references/cce-sle15-avail.txt , since there we keep the avalable CCE ids to use for SLE15 platform

Reply:
done in https://github.com/ComplianceAsCode/content/pull/10161/commits/8927598c575a04ebe58b807dbed0aee155971759

Reply:
Is it ok to just set it to resolved?